client/landing/gutenboarding/components/close-button/index.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Button } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+
+interface CloseButtonProps {
+	onClose: () => void;
+}
+
+const CloseButton = ( { onClose }: CloseButtonProps ) => {
+	const { __: NO__ } = useI18n();
+	return (
+		<Button onClick={ onClose } label={ NO__( 'Close dialog' ) }>
+			<svg
+				width="12"
+				height="12"
+				viewBox="0 0 16 16"
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+			>
+				<path d="M1.40456 1L15 15M1 15L14.5954 1" stroke="#1E1E1E" strokeWidth="1.5" />
+			</svg>
```

Comment:
Could maybe use the [tag icon](https://github.com/WordPress/gutenberg/blob/a97f31c4a80f4d945d10cfd6a7a5301baad9dabf/packages/icons/src/library/tag.js) from the `@wordpress/icons` npm

Reply:
https://github.com/Automattic/wp-calypso/pull/40742/files#r404025438